packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -294,6 +294,18 @@ export class BillingService {
     return;
   }
 
+  async provisionPreviewCustomer(workspaceId: string): Promise<null> {
+    if (!this.isBillingEnabled) {
+      return;
+    }
+
+    await this.stiggClient.provisionCustomer({
+      customerId: workspaceId,
+      shouldSyncFree: false,
+      subscriptionParams: { planId: BillingPlan.Enterprise }, // TODO: CHANGE IT TO PREVIEW PLAN
```

Comment:
How will we not forget this part (set a preview plan)? Perhaps add it to the Enum (BillingPlan) already if possible?

Reply:
the TODO is a keyword on my editor and this is also one of the reasons this pr is still a draft